content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -222,222 +206,240 @@ On Windows, you can use the following settings to configure Services and load ba
+#### Docker EE
 
-At a high level, these OS concepts are different:
+{{< feature-state for_k8s_version="v1.14" state="stable" >}}
 
-* Identity - Linux uses userID (UID) and groupID (GID) which are represented as integer types. User and group names are not canonical - they are an alias in `/etc/groups` or `/etc/passwd` back to UID+GID. Windows uses a larger binary security identifier (SID) which is stored in the Windows Security Access Manager (SAM) database. This database is not shared between the host and containers, or between containers.
-* File permissions - Windows uses an access control list based on SIDs, rather than a bitmask of permissions and UID+GID
-* File paths - convention on Windows is to use `\` instead of `/`. The Go IO libraries accept both types of file path separators. However, when you're setting a path or command line that's interpreted inside a container, `\` may be needed.
-* Signals - Windows interactive apps handle termination differently, and can implement one or more of these:
-  * A UI thread handles well-defined messages including WM_CLOSE
-  * Console apps handle ctrl-c or ctrl-break using a Control Handler
-  * Services register a Service Control Handler function that can accept SERVICE_CONTROL_STOP control codes
+[Docker EE](https://docs.mirantis.com/containers/v3.0/dockeree-products/dee-intro.html)-basic 19.03+ is available as a container runtime for all Windows Server versions. This works with the legacy dockershim adapter.
+
+See [Install Docker](https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/deploy-containers-on-server#install-docker) for more information.
+
+## Windows OS version compatibility {#windows-os-version-support}
+
+Windows has strict compatibility rules, where the host OS version must match the container base image OS version. Only Windows containers with a container operating system of Windows Server 2019 are supported.
+
+The following table shows Windows operating system compatibility with Kubernetes:
 
-Exit Codes follow the same convention where 0 is success, nonzero is failure. The specific error codes may differ across Windows and Linux. However, exit codes passed from the Kubernetes components (kubelet, kube-proxy) are unchanged.
+| Kubernetes version | Windows Server LTSC releases | Windows Server SAC releases |
+| --- | --- | --- |
+| *Kubernetes v1.17* | Windows Server 2019 | Windows Server ver 1809 |
+| *Kubernetes v1.18* | Windows Server 2019 | Windows Server ver 1809, Windows Server ver 1903, Windows Server ver 1909 |
+| *Kubernetes v1.19* | Windows Server 2019 | Windows Server ver 1909, Windows Server ver 2004 |
+| *Kubernetes v1.20* | Windows Server 2019 | Windows Server ver 1909, Windows Server ver 2004 |
```

Comment:
Is this maintained somewhere else? I believe we recently updated this list for v1.21

Reply:
I might have missed that in a rebase - let me check.

Reply:
If there's documentation we can link to, we can use that. I did previously consider moving this list of supported versions into the Reference section of the docs, and then linking there.

Reply:
the updates for this were merged yesterday: https://github.com/kubernetes/website/pull/26486